Makefile

```diff
@@ -130,6 +130,15 @@ vet:
 		exit 1; \
 	fi
 
+.PHONY: lint
+lint:
+ifeq (, $(shell which revive))
+	$(shell go install github.com/mgechev/revive)
+	$(info revive installed)
+endif
```

Comment:
Is this really a good idea to install it here without asking the user? Maybe we better invert the if and tell the user to install `revive`!?

Reply:
@srebhan good point, from a CI perspective this was easier but I can move the installation logic out and into the circleci config and just have it print a error message if it isn't installed.